scipy/stats/stats.py

```diff
@@ -6392,17 +6394,17 @@ def ks_1samp(x, cdf, args=(), alternative='two-sided', mode='auto'):
     >>> stats.ks_1samp(x, stats.norm.cdf, alternative='less')
     (0.12464329735846891, 0.040989164077641749)
 
-    Reject equal distribution against alternative hypothesis: less
+    Reject null hypothesis in favor of alternative hypothesis: less
 
     >>> stats.ks_1samp(x, stats.norm.cdf, alternative='greater')
     (0.0072115233216311081, 0.98531158590396395)
 
-    Don't reject equal distribution against alternative hypothesis: greater
+    Reject null hypothesis in favor of alternative hypothesis: less
```

Comment:
Should this still say 'greater:?

Reply:
Absolutely should! I thought I was careful with the copy-paste but apparently not. Thanks for checking. Fixed!